plugins/agent-harness-flight-recorder/scripts/evidence_index.py

```
@@ -200,6 +221,705 @@ class Chunk:
+    return config, key
+
+
+def _source_inventory(root: Path) -> dict[str, Any]:
+    strict_preflight(root)
+    receipts = load_receipts(root)
+    tracked = {
+        path for path in tracked_paths(root)
+        if CHUNK_PATH_RE.fullmatch(path) is not None
+    }
+    if set(receipts) != tracked:
+        raise VaultError("index seal source inventory is invalid; run a rebuild")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)